File: Jsoup/src/main/java/framework/utils/UIUtils.java
package framework.utils;


import android.content.ClipboardManager;
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.design.widget.Snackbar;
import android.view.View;
import android.view.inputmethod.InputMethodManager;
import android.widget.TextView;

import framework.App;


/**
 * by y on 2016/7/26.
 */
public class UIUtils {


    public static Context getContext() {
        return App.getInstance();
    }

    public static String[] getStringArray(int id) {
        return getContext().getResources().getStringArray(id);
    }

    public static void startActivity(Class<?> clz) {
        Intent intent = new Intent(getContext(), clz);
        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
        getContext().startActivity(intent);
    }

    public static void startActivity(Class<?> clz, Bundle bundle) {
        Intent intent = new Intent(getContext(), clz);
        intent.putExtras(bundle);
        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
        getContext().startActivity(intent);
    }

    public static void copyCL(TextView textView) {
        ClipboardManager cm = (ClipboardManager) getContext().getSystemService(Context.CLIPBOARD_SERVICE);
        cm.setText(textView.getText().toString().trim());
    }

    public static void offKeyboard() {
        InputMethodManager inputMethodManager = (InputMethodManager) getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
        inputMethodManager.toggleSoftInput(0, InputMethodManager.HIDE_NOT_ALWAYS);
    }

    public static void SnackBar(View view, Object object) {
        Snackbar.make(view, object + "", Snackbar.LENGTH_SHORT).show();
    }
}

File: sample_lock/src/main/java/framework/utils/UIUtils.kt
package framework.utils

import android.view.View
import com.google.android.material.snackbar.Snackbar

/**
 * by y on 2017/2/15
 */

object UIUtils {

    fun SnackBar(view: View, `object`: Any) {
        Snackbar.make(view, `object`.toString(), Snackbar.LENGTH_SHORT).show()
    }

    fun SnackBar(view: View, `object`: Any, color: Int) {
        Snackbar.make(view, `object`.toString(), Snackbar.LENGTH_SHORT)
            .setActionTextColor(color)
            .show()
    }

}
